remix-lib/package.json

```diff
@@ -23,7 +23,8 @@
     "web3": "^0.15.3",
     "solc": "^0.4.13",
     "standard": "^7.0.1",
-    "tape": "^4.6.0"
+    "tape": "^4.6.0",
+    "yo-yoify": "^4.3.0"
```

Comment:
do you need yo-yoify here?

Reply:
can  we remve it?

Reply:
@yann300 Sure. I'll do this.

Reply:
Strange, I don't have it in my repo 
![screen shot 2018-02-19 at 3 06 11 pm](https://user-images.githubusercontent.com/1076427/36384229-7c9a2372-1586-11e8-90ff-eb789f23bbdb.png)